clang-tools-extra/clangd/refactor/Rename.cpp

```diff
@@ -1308,7 +1308,7 @@ getMappedRanges(ArrayRef<Range> Indexed, ArrayRef<SymbolRange> Lexed) {
     return std::nullopt;
   }
   // Fast check for the special subset case.
-  if (std::includes(Indexed.begin(), Indexed.end(), Lexed.begin(), Lexed.end()))
+  if (llvm::includes(Indexed, Lexed))
```

Comment:
For ADT changes, we land the API changes separately from their uses/refactoring of existing code. ADT is tricky to get right and we often run into bugs in supported toolchains (like very specific versions of MSVC), so having API changes separately makes it easier to revert changes when this comes up.

Could you split this PR into two and add some very basic unit tests? Just to make sure the arguments are passed in the correct order etc.